spec/reek/smells/attribute_spec.rb

```diff
@@ -17,6 +17,47 @@
       ctx = Reek::Core::CodeContext.new(nil, Reek::Source::SourceCode.from(src).syntax_tree)
       expect(@detector.examine_context(ctx)).to be_empty
     end
+
+    shared_examples_for 'no attribute found' do
+      before :each do
+        ctx = Reek::Core::CodeContext.new(nil, Reek::Source::SourceCode.from(@src).syntax_tree)
+        @smells = @detector.examine_context(ctx)
+      end
+
+      it 'records no attribute' do
+        expect(@smells.length).to eq(0)
```

Comment:
Some of our specs use this old method of checking for smells, but we also have `.should_not reek_of(:Attribute)`, which reduces this whole shared example block to just one line, which can then be inlined.